doc/users/prev_whats_new/whats_new_1.0.rst

```diff
@@ -69,7 +69,7 @@ figures since the last call.  Eric has done a lot of testing on the
 user interface toolkits and versions and platforms he has access to,
 but it is not possible to test them all, so please report problems to
 the `mailing list
-<http://mail.python.org/mailman/listinfo/matplotlib users>`__
+<http://mail.python.org/mailman/listinfo/matplotlib-users>`__
```

Comment:
If http links bother you that much, at some point it'd be nice if you could go through the code and fix those. We are loosing a lot of time in going back on forth on that, and it is a very minor detail that bring little to no benefit to our documentation.